src/applications/login/containers/SignInApp.jsx

```diff
@@ -59,182 +72,77 @@ class SignInPage extends React.Component {
     const loggedOut = query.auth === 'logged_out';
 
     return (
-      <main className="login">
```

Comment:
You dropped this semantic tag I don't know if it's significant for screen readers or anything

Reply:
it's just in `SignInWrapper` now, which is a parent to both the landing page `/` route and the `/verify` route